packages/test-helpers/lib/web3.js

```diff
@@ -0,0 +1,130 @@
+module.exports = web3 => {
```

Comment:
Decide to use a single web3 helper file as we have in aragonOS. What do you think?